d2core/d2inventory/inventory_item_misc.go

```diff
@@ -0,0 +1,64 @@
+package d2inventory
+
+import (
+	"github.com/OpenDiablo2/OpenDiablo2/d2common/d2data/d2datadict"
+	"github.com/OpenDiablo2/OpenDiablo2/d2common/d2enum"
+	"log"
+)
+
+type InventoryItemMisc struct {
```

Comment:
Could we make this more generic and simply inherit properties? InventoryItemArmor and Weapon both use these fields as well, so it'd make sense to make a generic version and just extend for those.

Reply:
Could be, those 3 structs actually already implement an `InventoryItem` interface (d2core/inventory/inventory_item.go), how'd you suggest we make it more generic?

Reply:
Gotcha, I'll accept for now, we can clean it up later. Thanks again!